config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -41,19 +43,11 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-        imagePullPolicy: Always
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191105-e60677a-master
```

Comment:
we should go ahead and bump this, and this PR should go when we're ready to bump go